src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs

```diff
@@ -101,74 +112,70 @@ private static double GetMinFromBoundaries(MetricPoint pointData)
 						// - The average in the bucket (smallest if there are multiple positive measurements
 						// smaller than the lowest boundary)
 						return Math.Min(
-							Math.Min(pointData.ExplicitBounds[i], pointData.DoubleValue),
-							pointData.DoubleValue / pointData.LongValue);
+							Math.Min(buckets[i].ExplicitBound, sum),
```

Comment:
I think i remember why we needed this average here: If we would just take the sum of all values smaller than the lowest value, we might actually get a value that is bigger than the smallest bound (e.g. smallest bound is 1, and we record 3 measurements of 0.6, the sum would be 1.8, and using that over 1 would make no sense in this case)

Reply:
Ah yes, that makes sense of course!